### DISCUSSION TITLE
speed fan control

### DISCUSSION THREAD

**Discussion by jackdaw82** [DomiStyle/esphome-panasonic-ac#107] [General]
Is it possible change speed fan ??

**Comment by DomiStyle** [comment]
Yes.

**Reply by host-felice** [reply]
> Yes.

how? :-) I don't see the fan in the code

**Comment by pontusfischer** [comment]
Only the inside fan generally speaking. At least on my PZ35-WKE (CS-PZ35WKE + CU-PZ35WKE), the fan on the outside unit is either on or off. Unfortunately. Also this is not a configurable setting, the unit determines the fan state automatically, which probably makes sense.
